adapter/persistence/latest_entry_repository.go

```diff
@@ -0,0 +1,68 @@
+package persistence
+
+import (
+	"github.com/pkg/errors"
+	"github.com/utahta/momoclo-channel/domain"
+	"github.com/utahta/momoclo-channel/domain/entity"
+)
+
+type LatestEntryRepository struct {
+	DatastoreHandler
+}
+
+func NewLatestEntryRepository(h DatastoreHandler) *LatestEntryRepository {
+	return &LatestEntryRepository{h}
+}
+
+func (repo *LatestEntryRepository) Save(l *entity.LatestEntry) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method LatestEntryRepository.Save should have comment or be unexported